src/store/transactions/withdraw/withdraw.thunks.ts

```diff
@@ -180,6 +180,7 @@ function withdraw(
       global: { wallet, signer },
     } = getState();
     const withdrawalId = `${account.accountIndex}${exit.batchNum}`;
+    const alreadyDone = "WITHDRAW_ALREADY_DONE";
```

Comment:
Why write a message to be shown to the user uppercase and using underscores?

Reply:
Well, I know, it's like an error code. I would prefer a standard msg, but it's ok.

Reply:
the initial idea was show directly the message but comes with a lot of irrelevant info for the user, so I used the same string to match and show